settings-gradle.lockfile

```diff
@@ -1,4 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
why deleting this file?

Reply:
the file was not deleted, but it has been regenerated to add the new kafka references using 
`./gradlew dependencies --write-locks`

Reply:
I see it as deleted in this PR and no other file can be found here. This is not the `gradle.locking` file (that I see as modified as expected), this is a settings file that will be deleted by this pr

Reply:
my bad. 
this was definitelly a mistake, i have restored the file from the main branch here ce6fc59616681b275e7a5f770f8ea1b86a1dd628